src/main/java/io/debezium/connector/yugabytedb/YugabyteDBConnectorConfig.java

```diff
@@ -702,6 +702,14 @@ public JdbcConfiguration getJdbcConfig() {
             .withDescription(
                     "Whether or not to delete the logical replication stream when the connector finishes orderly" +
                             "By default the replication is kept so that on restart progress can resume from the last recorded location");
+        
+    public static final Field QL_TYPE = Field.create(DATABASE_CONFIG_PREFIX + "qltype")
+            .withDisplayName("The Query Language being used for tables to stream, in YugabyteDB (either ysql or ycql)")
+            .withType(Type.STRING)
+            .withImportance(Importance.MEDIUM)
+            .withDefault("ysql")
```

Comment:
Add this default value to the top of the class to a final variable `DEFAULT_QL_TYPE`